### ISSUE TITLE
Compiler warnings for OTP 27

### ISSUE THREAD

**Issue by big-r81** [apache/couchdb#5153]
Some compiler warnings emitted by OTP 26:

```
$ ./make

/couchdb/src/recon/src/recon_alloc.erl:702:7: Warning: matching on the float 0.0 will no longer also match -0.0 in OTP 27. If you specifically intend to match 0.0 alone, write +0.0 instead.
/couchdb/src/recon/src/recon_alloc.erl:702:11: Warning: matching on the float 0.0 will no longer also match -0.0 in OTP 27. If you specifically intend to match 0.0 alone, write +0.0 instead.

/couchdb/src/couch/src/couch_ejson_size.erl:35:14: Warning: matching on the float 0.0 will no longer also match -0.0 in OTP 27. If you specifically intend to match 0.0 alone, write +0.0 instead.
```

**Comment by pgj** [comment]
This has been addressed in `recon` upstream: https://github.com/ferd/recon/commit/c7c07ad8193ec6470c775c213e15e0f3447114b7

**Comment by big-r81** [comment]
> This has been addressed in `recon` upstream: [ferd/recon@c7c07ad](https://github.com/ferd/recon/commit/c7c07ad8193ec6470c775c213e15e0f3447114b7)

Thx for the hint Gábor. 🙏
